resources/troubleshooting-guides/push_notification_troubleshooting.md

```diff
@@ -0,0 +1,172 @@
+   3. In the `onComplete` function of `OnCompleteListener`, set the Push identifier by calling the API `MobileCore.setPushIdentifier`.  
+   For more information, see [setPushIdentifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics-mobile-services#set-up-push-messaging).
+   
+   #### Verify that the push token has successfully synced with the Experience Cloud ID service
+
+{% hint style="warning" %}
+If SDK privacy status is `optedout`, the push identifier will not be set.
+{% endhint %}
+
+   1. Verify the push token sync with the Experience Cloud ID service (ECID).  
+   2. To verify, launch your app connected to a Project Griffon session.   
+   3. In the list of events, verify that you have an event with type _SetPushIdentifier_.
+   4. In the details panel on the right, verify the value of the push token for this device.  
+      The value in pushIdentifier is the same value that is sent to the ECID service. For more information, see screenshot   below.  
+      
+![Verify SetPushIdentifier event](../../.gitbook/assets/push_token_to_identity.png "Verify SetPushIdentifier event.")
+
+   5. To verify that app’s push token is mapped to the correct Experience cloud ID(ECID) in the Campaign instance, click  **_Adobe Campaign -> Administrator -> Channels -> Mobile App (AEP SDK)_**.  
+   6. Select your app, under the mobile application subscribers verify that the Experience Cloud ID and the Registration token for the user is displayed.  
+   
+   ![App subscriber list, verify mid and push token](../../.gitbook/assets/subscriber_list_android.png "App subscriber list, verify mid and push token.")  
+   
+   6. If you are using Charles, verify that the push token has successfully synced with the ECID service.  
+   7. Check for the _demdex request_, which is marked with the red line in the screenshot below.  
+   8. Verify the successful response(200) for this network call.       
+ 
+ ![ECID network request for push token sync](../../.gitbook/assets/push_identifier.png "ECID network request for push token sync.")
+ 
+ {% endtab %}
+
+{% tab title="iOS" %}
+### Set up the iOS app  
+  
+  1. After you launch the app, call `registerForRemoteNotifications` to register the app and receive the APNS token.  
+  This API generates and returns the APNS token through the `application:didRegisterForRemoteNotificationsWithDeviceToken:` delegate function.  
+  For more information, see [Registering Your App with APNs](https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns?language=objc)
+  2. In `application:didRegisterForRemoteNotificationsWithDeviceToken:`, call `setPushIdentifier` to set the Push identifier.  
+  For more information, see [setPushIdentifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics-mobile-services#set-up-push-messaging).
+  
+  #### Verify that the push token has successfully synced with the Experience Cloud ID service
```

Comment:
Please move this heading flush left, otherwise it will not display as a heading.